go.mod

```diff
@@ -111,3 +111,5 @@ replace (
 
 	kubevirt.io/client-go => ./staging/src/kubevirt.io/client-go
 )
+
+go 1.13
```

Comment:
can we remove this from the commit

Reply:
I guess this is automatically added by vscode.